.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+sudo: required
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+install:
+  - sudo rm -vf /etc/apt/sources.list.d/*
```

Comment:
Why do we need this?

Reply:
This refers to an issue of travis CI https://github.com/travis-ci/travis-ci/issues/8607.
I don't think we need this if we remove sudo.